lib/sinon/sandbox.js

```diff
@@ -196,18 +196,15 @@ function Sandbox() {
     };
 
     sandbox.restoreContext = function restoreContext() {
-        var injectedKeys = sandbox.injectedKeys;
-        var injectInto = sandbox.injectInto;
-
-        if (!injectedKeys) {
+        if (!sandbox.injectedKeys) {
```

Comment:
Is this optimization needed? `forEach` can handle empty arrays.

Reply:
I don't know if it is, but it was there already.